cluster/gce/gci/configure-helper.sh

```diff
@@ -997,6 +997,8 @@ function start-node-problem-detector {
   flags+=" --logtostderr"
   flags+=" --system-log-monitors=${km_config},${dm_config}"
   flags+=" --apiserver-override=https://${KUBERNETES_MASTER_NAME}?inClusterConfig=false&auth=/var/lib/node-problem-detector/kubeconfig"
+  local -r npd_port=${NODE_PROBLEM_DETECTOR_PORT:-20256}
```

Comment:
Not entirely sure, but I would like to surface the port allocation better to avoid other collisions in the future...